mount.h

```diff
@@ -366,7 +366,7 @@ class Mount : public wxMessageBoxProxy
     virtual wxString CalibrationSettingsSummary() const { return wxEmptyString; }
 
     virtual bool CalibrationFlipRequiresDecFlip();
-    virtual bool HasHPDecEncoder() { return false; }
+    virtual bool HasHPEncoders() { return false; }

```

Comment:
let's make this a const method (indicating that calling it has no side-effects on the state of the object)
```suggestion
    virtual bool HasHPEncoders() const { return false; }
```